tasks/dependencies.yml

```diff
@@ -88,7 +85,6 @@
 
 - name: Install app specifc dependencies
   yum:
-    name: "{{ item }}"
+    name: "{{ rails_app_additional_dependencies | default([]) }}"
```

Comment:
I'm worried about this one so will wait to get @bibliotechy's input.  Wouldn't this be a string "[]" vs an actual array []?

Reply:
@dkinzer I think that is how it is supposed to be, I just moved what we already had on line 93.  It is written with the quotes there too.

Reply:
But we can wait for Chad if you want.

Reply:
Hmm.. OK, I guess if everything is treated as text.